addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19023,7 +19023,13 @@ msgctxt "#35270"
 msgid "Your account is not verified. Please check your email to complete your sign up."
 msgstr ""
 
-#empty strings from id 35271 to 35504
+#. Label for showing which emulator the game is playing with
+#: system/settings/settings.xml
+msgctxt "#35271"
+msgid "Playing with:"
```

Comment:
Perhaps just me, but `with` also seems an unnecessary addition to me, why not just have `Playing` here? the simpler you can keep it the more likey it can be reused.